include/freerdp/settings.h

```diff
@@ -1697,6 +1697,7 @@ extern "C"
 	FREERDP_API void freerdp_target_net_addresses_free(rdpSettings* settings);
 	FREERDP_API BOOL freerdp_target_net_addresses_copy(rdpSettings* settings, char** addresses,
 	                                                   UINT32 count);
+	FREERDP_API BOOL proxy_parse_uri(rdpSettings* settings, const char* uri_in);
```

Comment:
Should be in `include/freerdp/utils/proxy_utils.h` or similar , not really settings